settings.go

```diff
@@ -462,359 +463,360 @@ func (s *SettingsService) GetSettings(options ...RequestOptionFunc) (*Settings,
+	ContainerRegistryImportMaxRetries                     *int                      `url:"container_registry_import_max_retries,omitempty" json:"container_registry_import_max_retries,omitempty"`
+	ContainerRegistryImportMaxStepDuration                *int                      `url:"container_registry_import_max_step_duration,omitempty" json:"container_registry_import_max_step_duration,omitempty"`
+	ContainerRegistryImportMaxTagsCount                   *int                      `url:"container_registry_import_max_tags_count,omitempty" json:"container_registry_import_max_tags_count,omitempty"`
+	ContainerRegistryImportStartMaxRetries                *int                      `url:"container_registry_import_start_max_retries,omitempty" json:"container_registry_import_start_max_retries,omitempty"`
+	ContainerRegistryImportTargetPlan                     *string                   `url:"container_registry_import_target_plan,omitempty" json:"container_registry_import_target_plan,omitempty"`
+	ContainerRegistryTokenExpireDelay                     *int                      `url:"container_registry_token_expire_delay,omitempty" json:"container_registry_token_expire_delay,omitempty"`
+	CustomHTTPCloneURLRoot                                *string                   `url:"custom_http_clone_url_root,omitempty" json:"custom_http_clone_url_root,omitempty"`
+	DNSRebindingProtectionEnabled                         *bool                     `url:"dns_rebinding_protection_enabled,omitempty" json:"dns_rebinding_protection_enabled,omitempty"`
+	DSAKeyRestriction                                     *int                      `url:"dsa_key_restriction,omitempty" json:"dsa_key_restriction,omitempty"`
+	DeactivateDormantUsers                                *bool                     `url:"deactivate_dormant_users,omitempty" json:"deactivate_dormant_users,omitempty"`
+	DeactivateDormantUsersPeriod                          *int                      `url:"deactivate_dormant_users_period,omitempty" json:"deactivate_dormant_users_period,omitempty"`
+	DecompressArchiveFileTimeout                          *int                      `url:"decompress_archive_file_timeout,omitempty" json:"decompress_archive_file_timeout,omitempty"`
+	DefaultArtifactsExpireIn                              *string                   `url:"default_artifacts_expire_in,omitempty" json:"default_artifacts_expire_in,omitempty"`
+	DefaultBranchName                                     *string                   `url:"default_branch_name,omitempty" json:"default_branch_name,omitempty"`
+	DefaultBranchProtection                               *int                      `url:"default_branch_protection,omitempty" json:"default_branch_protection,omitempty"`
+	DefaultBranchProtectionDefaults                       *BranchProtectionDefaults `url:"default_branch_protection_defaults,omitempty" json:"default_branch_protection_defaults,omitempty"`
```

Comment:
This field should be set to a `BranchProtectionDefaultsOptions` struct which contains the required fields as pointers (see other option structs).

Reply:
This should be addressed now.